registry/proxy/proxyauth.go

```diff
@@ -19,7 +19,7 @@ type credentials struct {
 }
 
 func (c credentials) Basic(u *url.URL) (string, string) {
-	up := c.creds[u.String()]
+	up := c.creds[tokenURL]
```

Comment:
@dmcgowan Why are we limiting the use of token auth here? Seems like the wrong place to be making that determination. Should we add a comment noting this limitation?